src/test/java/com/wikia/webdriver/common/templates/NewTestTemplate.java

```diff
@@ -43,12 +41,34 @@ private boolean isTestExcludedFromEnv(Method method){
   @BeforeMethod(alwaysRun = true)
   public void start(Method method, Object[] data) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "start" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)